crates/kira/src/sound/static_sound/data.rs

```diff
@@ -408,6 +401,11 @@ impl StaticSoundData {
 			},
 		)
 	}
+
+	/// Returns the sample rate of the audio.
```

Comment:
I don't understand the benefit of this function when `sample_rate` is already a public field.

Reply:
I just find it weird as a crate consumer that both sound data types do not have exact same api, but here you're right it's probably redundant, I'm gonna remove it.